<launch>
  <node name="hsv_filtering" pkg="chinorobo_ros_usbcam_02" type="hsv_filter_node.py">
    <remap from="image" to="/usb_cam/image_raw" />
  </node>
  <node name="reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />
  <node name="filtered_view" pkg="image_view" type="image_view">
    <remap from="image" to="/hsv_filter/image_filtered"/>
  </node>
  <node name="mask_view" pkg="image_view" type="image_view">
    <remap from="image" to="/hsv_filter/image_mask"/>
  </node>
</launch>